components/content/AdaptationCard.tsx

```
@@ -26,13 +32,11 @@ const AdaptationCard: React.FC<AdaptationCardProps> = ({
         <p>{original}</p>
         
         <div className={workflowStyles.title}>{platform} Adaptation:</div>
-        <div dangerouslySetInnerHTML={{ __html: adaptation }} />
-      </div>
-      <div className={workflowStyles.notes}>
+        <div dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(adaptation) }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)